src/services/API/Item.js

```diff
@@ -0,0 +1,24 @@
+import { apiClient } from "@/services/API/RootClient.js";
+
+export default {
+  createItem(itemDetails) {
+    // API to create an item in the DB
+    // converting time from seconds to milliseconds
+    itemDetails.time = itemDetails.time * 1000;
+    return apiClient()
+      .post(process.env.VUE_APP_BACKEND_ITEMS, itemDetails)
+      .then((response) => {
+        response.data.time = response.data.time / 1000;
+        return response.data;
+      });
+  },
+
+  updateItem(itemDetails) {
+    // API to update an item in the DB
+    itemDetails.time = itemDetails.time * 1000;
+    return apiClient().put(
```

Comment:
so there are two types of updates DRF provides:
1. Partial update (uses PATCH)
2. Update (uses PUT)

If we're just updating a few fields (and not all), maybe we can consider the first one.

For example, like below, for session-answer, I'm just updating the answer column and nothing else using "PATCH".
![image](https://user-images.githubusercontent.com/12053186/113873936-e96b9400-97d2-11eb-8111-76c3be6497ae.png)

Reply:
Makes sense. My thinking was that in this function, both the things are possible.
Updating the whole schema or just one field
Should I go ahead and create a separate function, something like `partialUpdateItem`?
Or pass an argument here like `isPartial` and the request method switches according to that?

Reply:
I think what we need to do is just use PUT and not PATCH in all update functions for now - this is an optimization which we can return to later - can be made an issue - seems very low priority to me.